modules/atomic-widgets/widgets/atomic-heading.php

```diff
@@ -98,6 +98,7 @@ protected static function define_props_schema(): array {
 				->default( 'h2' ),
 
 			'title' => String_Prop_Type::make()
+//				->meta( Dynamic_Tags::ignore() )
```

Comment:
```suggestion
//				->add_meta( Dynamic_Tags::ignore() )
```
??

Reply:
Done
even though I think it's probably fine because it's the same convention like other methods 🤷